api/ImageData.json

```diff
@@ -0,0 +1,286 @@
+{
+  "api": {
+    "ImageData": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ImageData",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": true
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "14"
+          },
+          "firefox_android": {
+            "version_added": "14"
+          },
+          "ie": {
+            "version_added": "9"
+          },
+          "opera": {
+            "version_added": "9"
+          },
+          "opera_android": {
+            "version_added": "10"
+          },
+          "safari": {
+            "version_added": "3.1"
+          },
+          "safari_ios": {
+            "version_added": "3.2"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "Support in workers": {
```

Comment:
No spaces in identifiers. Please name this `worker_support` and add `"description": "Support in workers",`.

Reply:
👍